src/paperoni/refinement/pdf/model.py

```diff
@@ -24,6 +24,10 @@
 - Check Completeness:
   - Ensure no author is omitted from the list.
   - Ensure all affiliations are listed correctly for each author.
+- Normalize these affiliation names. The name in quotes is the normalized name that you should use.
+  - "Mila": could be listed as MILA, Montreal Institute for Learning Algorithms, IQIA, Institut Québécois d'Intelligence Artificielle
+  - "Université de Montréal": could be listed in English as University of Montreal, possibly as the shortened UdeM
```

Comment:
One thing we don't know yet I believe is how the model will react when Mila - Université de Montréal and Mila and Université de Montréal will be listed as different affiliations. I don't think it's the norm so it might not be much of an issue but I've seen this at least once